### ISSUE TITLE
FUSE xattr code broken on Linux

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#1282]
borg mount crashes on Linux.

```
# ls -l /mnt
ls: /mnt: Input/output error
ls: cannot open directory /mnt: Transport endpoint is not connected
```

When running borg mount -f, the real problem gets visible:
```
$ ls -ld /mnt
File "/home/askalski/private/src/borg/borg/fuse.py", line 197, in getxattr
    raise llfuse.FUSEError(errno.ENOATTR) from None
AttributeError: module 'errno' has no attribute 'ENOATTR'
```
Reason:

After the initial fix for FUSE xattr code on OS X, it is now broken on Linux (in borg 1.0.4), because OS X Python has errno.ENOATTR, but Linux Python does not. :-(

The llfuse documentation just said that ENOATTR needs to be returned, but did not point out that it means llfuse.ENOATTR and *not* errno.ENOATTR.

**Comment by ThomasWaldmann** [comment]
Workaround: use 1.0.3 for borg mount (if not on OS X) or do not access not existing xattr keys.

**Comment by ThomasWaldmann** [comment]
@LocutusOfBorg ^ guess there will be a 1.0.5 very soon due to that, so maybe just skip 1.0.4.

**Comment by LocutusOfBorg** [comment]
If it is a matter of hours... we can wait :) thanks

**Comment by ThomasWaldmann** [comment]
Fixed in 1.0-maint (and soon in 1.0.5).
